src/lib/util/cron.test.js

```diff
@@ -0,0 +1,50 @@
+import { normalize, format } from "./cron";
+
+describe("cron utils", () => {
+  describe("normalize", () => {
+    it("should convert non-standard expressions to standard format", () => {
+      expect(normalize("@yearly")).toEqual("0 0 0 1 1 *");
+      expect(normalize("@annually")).toEqual("0 0 0 1 1 *");
+      expect(normalize("@monthly")).toEqual("0 0 0 1 * *");
+      expect(normalize("@daily")).toEqual("0 0 0 * * *");
+      expect(normalize("@midnight")).toEqual("0 0 0 * * *");
+      expect(normalize("@hourly")).toEqual("0 0 * * * *");
+
+      expect(normalize("@every 48h")).toEqual("0 0 0 */2 * *");
+      expect(normalize("@every 5h")).toEqual("0 0 */5 * * *");
+      expect(normalize("@every 5m")).toEqual("0 */5 * * * *");
+      expect(normalize("@every 1h5m")).toEqual("0 */65 * * * *");
+      expect(normalize("@every 5s")).toEqual("*/5 * * * * *");
+
+      expect(normalize("not a valid expression")).toEqual(
+        "not a valid expression",
+      );
+
+      expect(() => normalize("@every not a valid duration")).toThrow();
+    });
+  });
+
+  describe("format", () => {
+    it("should correctly format non-standard expressions", () => {
+      expect(format("@yearly")).toEqual(
+        "At 12:00 AM, on day 1 of the month, only in January",
+      );
+      expect(format("@annually")).toEqual(
+        "At 12:00 AM, on day 1 of the month, only in January",
+      );
+      expect(format("@monthly")).toEqual("At 12:00 AM, on day 1 of the month");
+      expect(format("@daily")).toEqual("At 12:00 AM");
+      expect(format("@midnight")).toEqual("At 12:00 AM");
+      expect(format("@hourly")).toEqual("Every hour");
+
+      expect(format("@every 5s")).toEqual("Every 5 seconds");
```

Comment:
For my sanity could you add a case for the `5ms`? I would just like to see us cover the terminal case.